src/app/zap-templates/zcl/data-model/chip/localization-unit-cluster.xml

```diff
@@ -29,9 +29,18 @@ limitations under the License.
 may have differing preferences for the units in which values are conveying in communication to a
 user. As such, Nodes that visually or audibly convey measurable values to the user need a
 mechanism by which they can be configured to use a user’s preferred unit.</description>
+<attribute side="server" code="0x0000" define="TEMPERATURE_UNIT" type="TempUnitType" writable="true" optional="true">TemperatureUnit</attribute>
```

Comment:
Again, why is this not TempUnit like in the spec?

Reply:
I was making then consistent, I set it back.